e2e/getNightfallBalances.test.ts

```diff
@@ -0,0 +1,18 @@
+import { Client } from "../libs/client";
+import testVars from "./config";
+
+describe("Verify client API get get nightfall balance", () => {
+  jest.setTimeout(15 * 100000);
+
+  it("Shoud return the address balance if the client is running", async () => {
+    const client = new Client(process.env.APP_CLIENT_API_URL);
+    const zkpKeys = await client.generateZkpKeysFromMnemonic(
+      testVars.NIGHTFALL_MNEMONIC,
+      0,
+    );
+
+    const balance = await client.getNightfallBalances(zkpKeys);
+    console.log(balance);
+    expect(balance).toBeTruthy();
+  });
+});
```

Comment:
Again I don't think this is an e2e test. In the SDK, an e2e test should at least use the `User` class. And to be honest this kind of integration test belongs to the client API.